src/components/Auth/AuthConfig.js

```diff
@@ -0,0 +1,173 @@
+import { loader } from './assets/loader'
+import { success } from './assets/success'
+import { borderRadius, color } from '@onfido/castor'
+
+export const Config = (function () {
```

Comment:
Is there any specific reason that this component wasn't written in TypeScript (while most of the others are)?

Reply:
Previous use-case provided by FaceTec was using a js file, and transpiling it to TS was not working 😒  I've since fixed this and will upload the `.tsx` version of this file, with additional changes to cater to custom UI provided by the client.